assets/javascripts/kitten/components/cards/engagement-card/index.js

```diff
@@ -70,15 +74,14 @@ export const EngagementCard = ({
 }
 
 EngagementCard.propTypes = {
-  imageSrc: PropTypes.string,
+  icon: PropTypes.node.isRequired,
```

Comment:
```suggestion
  icon: PropTypes.element.isRequired,
```
`element` est plus spécifique à un composant (https://www.npmjs.com/package/prop-types)